app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -48,94 +48,108 @@ fun Home(
 
     when {
         trendingNowMediaList != null && popularThisSeasonMediaList != null -> {
-            Box(
-                modifier = Modifier
-                    .verticalScroll(rememberScrollState())
-                    .navigationBarsPadding()
-            ) {
-                Box {
-                    Image(
-                        painter = painterResource(id = Res.drawable.background),
-                        contentDescription = "Background",
-                        modifier = Modifier.fillMaxWidth()
-                    )
-
-                    Text(
-                        text = "おかえり",
-                        color = Text,
-                        fontSize = 57.sp,
-                        modifier = Modifier
-                            .align(Alignment.BottomStart)
-                            .padding(start = 24.dp, bottom = (24 + 18).dp)
-                    )
-                }
-
-                // TODO: Use `verticalArrangement` instead of the `Spacer`s.
-                Column {
-                    Spacer(
-                        modifier = Modifier.size((LocalConfiguration.current.screenWidthDp - 18).dp)
-                    )
-
-                    Column(
-                        modifier = Modifier
-                            .clip(backdropShape)
-                            .background(Backdrop)
-                    ) {
-                        Spacer(modifier = Modifier.size(24.dp))
-
-                        Text(
-                            text = "Trending Now",
-                            color = Text,
-                            fontSize = 14.sp,
-                            fontFamily = manropeFamily,
-                            fontWeight = FontWeight.Bold,
-                            modifier = Modifier.padding(start = 24.dp)
+            AnimiteTheme {
+                Box(
+                    modifier = Modifier
+                        .verticalScroll(rememberScrollState())
+                        .navigationBarsPadding()
+                ) {
+                    Box {
+                        Image(
+                            painter = painterResource(id = Res.drawable.background),
+                            contentDescription = "Background",
```

Comment:
If the image is not important for accessibility, consider setting contentDescription to null.
Does the API provide a description of the image to use?

Reply:
I chose to use this image. There are images that can potentially use Kdocs generated by apollo-kotlin. I'll have to check if there are APIs for getting the Kdocs.